### PR TITLE
Added WordPress to list of GPL repos on homepage.

### PR DESCRIPTION
**Pull request by veganstraightedge**
WordPress is a very popular, very public facing open source project that uses GPL.

### CODE REVIEW

**Comment by cobyism** [comment]
:thumbsup: :sparkles: Good suggestion. Thanks!

File: index.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
       The GPL (<a href="licenses/gpl-v2">V2</a> or <a href="licenses/gpl-v3">V3</a>) is a copyleft license that requires others who modify your code to disclose their changes if they redistribute it in source or binary form. V3 is similar to V2, but further restricts use in hardware that forbids software alterations.
     </p>
     <p>
-      <strong>Linux</strong> and <strong>Git</strong> use the GPL.
+      <strong>Linux</strong>, <strong>Git</strong> and <strong>WordPress</strong> use the GPL.
```
Comment (haacked) [inline index.html:42]: Oxford commas please! :stuck_out_tongue:

Comment (cobyism) [inline index.html:42]: Derp, sorry I missed that. I’ll sort that out in a second.

Comment (cobyism) [inline index.html:42]: … and it looks like you beat me to it. :laughing:

**Pull request merged**